GramAddict/core/utils.py

```diff
@@ -207,6 +207,7 @@ def save_crash(device):
 
 
 def detect_block(device):
+    logger.debug("Detect block!")
```

Comment:
Maybe `Checking for block...` or `Checking for block!` This language is ambiguous and could make people think it detected a block.